added Query & resolvers for Random Quote and Get Quote By ID

KaleseCarpenter opened the pull request:
Hello, 

Please review when you have time. Thank You. 🙂 @librairica

KaleseCarpenter committed:
added schema

KaleseCarpenter committed:
regenerated models and schema

KaleseCarpenter committed:
Deleted schema.graphql file

KaleseCarpenter committed:
added gql schema

KaleseCarpenter committed:
deleted gql schema

KaleseCarpenter committed:
added comment

KaleseCarpenter committed:
Merge branch 'gqlgen' of https://github.com/KaleseCarpenter/quotes-starter into gqlgen

KaleseCarpenter committed:
added dummy data, query working

KaleseCarpenter committed:
updated schema and query resolver

KaleseCarpenter committed:
added getByID

KaleseCarpenter committed:
added comments, small changes

librairica reviewed (commented):
I see some merge conflicts in this branch. Let me know if you need help resolving them!

KaleseCarpenter committed:
schema updates

KaleseCarpenter committed:
added query resolver, fixed merge conflicts

KaleseCarpenter commented:
Ok I believe it is fixed. Thank so much for you help. The files that I needed reviewed are the last 3 above your comment and one below: 
- updated schema and query resolver 
-added get By ID
-added comments small changes
-schema update

librairica commented on gqlgen/graph/.txt:
```diff
@@ -0,0 +1,47 @@
+	}
+	return responseObject, nil
+
+	/*quote := &model.Quote{
+		ID:     "12-3422-1-23432-009954",
+		Quote:  "I love being my authentic self",
+		Author: "Kalese Carpenter",
+	}
+	return quote, nil*/
```
I totally understand wanting to hold onto the mock, but I recommend deleting comments when possible.

librairica commented on gqlgen/graph/schema.graphqls:
```diff
@@ -2,21 +2,18 @@
 # https://gqlgen.com/getting-started/
 
-type Quotes{
+type Quote {
```
Thanks for changing to singular, I think this is more intuitive.

librairica commented on gqlgen/graph/schema.graphqls:
```diff
@@ -2,21 +2,18 @@
   quote: String!
   "Author is the creator of the quote"
-  author: Author!
+  author: String!
 }
 
-"Author of a Quote"
-type Author {
-"The id of the author"
-  id: ID!
-  "The name of the person who created the quote"
-  name: String!
-}
```
Great simplification.

librairica commented on gqlgen/graph/schema.resolvers.go:
```diff
@@ -2,3 +2,76 @@ package graph
+func (r *Resolver) Query() generated.QueryResolver { return &queryResolver{r} }
+
+type queryResolver struct{ *Resolver }
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//    it when you're done.
+//  - You have helper methods in this file. Move them out to keep these resolver files clean.
```
```suggestion
```

librairica reviewed (approved):
<img width="1159" alt="Screen Shot 2022-10-16 at 6 54 23 PM" src="https://user-images.githubusercontent.com/56241711/196062337-adaf795a-6da3-4143-a8f3-cf0427b1bc45.png">
This is great! I left some suggestions, but the functionality is spot on.
